pkg/v1/remote/transport/error_test.go

```diff
@@ -89,6 +89,11 @@ func TestCheckErrorNotError(t *testing.T) {
 		code: http.StatusBadRequest,
 		body: "",
 		msg:  "unsupported status code 400",
+	}, {
```

Comment:
What about a test case for invalid JSON, to exercise the point you make in the comment.

Reply:
That's the next case, with a body of `Not JSON`.

Reply:
_How convenient_.